convex/sources.ts

```diff
@@ -39,50 +38,11 @@ async function addSource(
     )
   );
   await db.patch(sourceId, { chunkIds });
-  return (await db.get(sourceId))!;
+  return {
+    source: (await db.get(sourceId))!,
+  }!;
 }
 
-// Insert the source into the DB, along with the associated chunks.
-export const add = mutation({
-  args: { name: v.string(), chunks: v.array(InputChunk) },
-  handler: async (ctx, { name, chunks }) => {
-    const source = await addSource(ctx.db, name, chunks);
-    await ctx.scheduler.runAfter(0, internal.sources.addEmbedding, {
-      source,
-      texts: chunks.map(({ text }) => text),
-    });
-  },
-});
-
-// Make embeddings for a source's chunks and store them.
-export const addEmbedding = internalAction({
-  handler: async (
-    ctx,
-    { source, texts }: { source: Doc<"sources">; texts: string[] }
-  ) => {
-    const { embeddings, embeddingMs, totalTokens } = await fetchEmbeddingBatch(
-      texts
-    );
-    console.log({
-      batchSize: texts.length,
-      totalTokens,
-      embeddingMs,
-    });
-    await upsertVectors(
-      "chunks",
-      source.chunkIds.map((id, chunkIndex) => ({
-        id,
-        values: embeddings[chunkIndex],
-        metadata: { sourceId: source._id, textLen: texts[chunkIndex].length },
-      }))
-    );
-    await ctx.runMutation(internal.sources.patch, {
-      id: source._id,
-      patch: { saved: true, totalTokens, embeddingMs },
-    });
-  },
-});
-
```

Comment:
removed for now? It should be a matter of taking out the pinecone upsert and writing the embedding to a table instead